scripts/truncate_old_tables.py

```diff
@@ -0,0 +1,249 @@
+#! /usr/bin/env python3.8
+import datetime
+import logging
+import os
+import sys
+from decimal import Decimal
+
+import psycopg2
+from app_common_python import LoadedConfig
+from psycopg2.extras import NamedTupleCursor
+
+
+logging.basicConfig(
+    format="truncate_old_tables (%(process)d) :: %(asctime)s: %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+    level=getattr(logging, os.environ.get("KOKU_LOG_LEVEL", "INFO")),
+)
+LOG = logging.getLogger("truncate_old_tables")
+
+
+def connect():
+    engine = "postgresql"
+    app = os.path.basename(sys.argv[0])
+
+    if bool(os.environ.get("DEVELOPMENT", False)):
+        user = os.environ.get("DATABASE_USER")
+        passwd = os.environ.get("DATABASE_PASSWORD")
+        host = os.environ.get("POSTGRES_SQL_SERVICE_HOST")
+        port = os.environ.get("POSTGRES_SQL_SERVICE_PORT")
+        db = os.environ.get("DATABASE_NAME")
+    else:
+        user = LoadedConfig.database.username
+        passwd = LoadedConfig.database.password
+        host = LoadedConfig.database.hostname
+        port = LoadedConfig.database.port
+        db = LoadedConfig.database.name
+
+    url = f"{engine}://{user}:{passwd}@{host}:{port}/{db}?sslmode=prefer&application_name={app}"
+    LOG.info(f"Connecting to {db} at {host}:{port} as {user}")
+
+    # return psycopg2.connect(url, cursor_factory=RealDictCursor)
+    return psycopg2.connect(url, cursor_factory=NamedTupleCursor)
+
+
+def _execute(conn, sql, params=None):
+    cur = conn.cursor()
+    LOG.debug(cur.mogrify(sql, params).decode("utf-8"))
+    cur.execute(sql, params)
+    return cur
+
+
+def validate_tables(conn):
+    accounts = sorted(
```

Comment:
Instead of hard-coding these account numbers, can we make use of this function instead?
https://github.com/project-koku/koku/blob/1240bb93c0acee51dd9b255c73ddf1d91b4de449/koku/masu/processor/__init__.py#L19

Reply:
Done. That was a good catch. I forgot enablement went to the source-level.